desktop/libs/azure/src/azure/abfs/upload.py

```diff
@@ -13,36 +13,38 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from future import standard_library
-standard_library.install_aliases()
-import logging
+
 import sys
+import logging
 import unicodedata
 
-if sys.version_info[0] > 2:
-  from io import StringIO as string_io
-else:
-  from cStringIO import StringIO as string_io
-
 from django.core.files.uploadedfile import SimpleUploadedFile
 from django.core.files.uploadhandler import FileUploadHandler, SkipFile, StopFutureHandlers, StopUpload, UploadFileException
-from desktop.conf import TASK_SERVER
+from future import standard_library
 
-from desktop.lib.exceptions_renderable import PopupException
-from desktop.lib.fsmanager import get_client
 from azure.abfs.__init__ import parse_uri
 from azure.abfs.abfs import ABFSFileSystemException
+from desktop.conf import TASK_SERVER_V2
+from desktop.lib.exceptions_renderable import PopupException
+from desktop.lib.fsmanager import get_client
+from filebrowser.utils import calculate_total_size, generate_chunks
+
+standard_library.install_aliases()
```

Comment:
what is the use of "install_aliases"?

Reply:
It was already present in [https://github.com/cloudera/hue/blob/master/desktop/libs/azure/src/azure/abfs/upload.py#L16C1-L18C15](https://github.com/cloudera/hue/blob/master/desktop/libs/azure/src/azure/abfs/upload.py#L16C1-L18C15). I had to move `standard_library.install_aliases()` because ruff wants you to have all the imports statements in top of the file.

Reply:
Can remove this and line 23 future import